### PR TITLE
Turn on validation for read by default

### PR DESCRIPTION
**Pull request by benkiel**

### CODE REVIEW

**Comment by anthrotype** [comment]
Didn’t we say we want it on by default for both read and write?

**Comment by benkiel** [comment]
@anthrotype It was on by default for Write, but off for Read. This PR should turn it on for read now.

**Comment by anthrotype** [comment]
Oh OK then

File: Lib/ufoLib/glifLib.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def __init__(self, dirName, glyphNameToFileNameFunc=None, ufoFormatVersion=3, va
 		(including the .glif extension). The glyphNameToFileName function
 		is called whenever a file name is created for a given glyph name.
 
-		``validateRead`` will validate read operations. It's default is ``False``.
+		``validateRead`` will validate read operations. It's default is ``True``.
```
Comment (anthrotype) [inline Lib/ufoLib/glifLib.py]: “Its default is”, not “It’s default is”

**Pull request merged**